jobs/openvpn-client/templates/etc/profile.ovpn.erb

```diff
@@ -12,7 +12,9 @@ mute-replay-warnings
 
 nobind
 resolv-retry infinite
-
+<% if_p('username') do | v | %>
+auth-user-pass /var/vcap/jobs/openvpn-client/etc/user
```

Comment:
I haven't configured username/password authentication in a long time. Is it mutually exclusive with client certificates (lines 49 - 55); or can both be used; or are both required?

Reply:
It is an either or situation. My understanding is that both can be used together or they can be used separately.